presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -1893,6 +1907,77 @@ VeloxQueryPlanConverterBase::toVeloxQueryPlan(
       toVeloxQueryPlan(node->source, tableWriteInfo, taskId));
 }
 
+namespace {
+
+core::WindowNode::Function makeRowNumberFunction(
+    const protocol::VariableReferenceExpression& rowNumberVariable) {
+  core::WindowNode::Function function;
+  function.functionCall = std::make_shared<core::CallTypedExpr>(
+      stringToType(rowNumberVariable.type),
+      std::vector<core::TypedExprPtr>{},
+      "presto.default.row_number");
+
+  function.frame.type = core::WindowNode::WindowType::kRows;
```

Comment:
Just confirming that this is Rows and not Range since the default frame is between Range unbounded preceding and current row.

Reply:
This shouldn't matter since row_number() function doesn't use frame information.